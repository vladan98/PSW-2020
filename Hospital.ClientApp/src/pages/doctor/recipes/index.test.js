import Recipes from "."
import render from "../../../helpers/renderComponent"

describe("Recipes", () => {
    it("should loading spinner and recipes form", () => {
        const { container } = render(<Recipes />)

        expect(container).toMatchSnapshot()
    })
})