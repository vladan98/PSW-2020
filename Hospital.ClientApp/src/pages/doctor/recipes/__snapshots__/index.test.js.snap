// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipes should loading spinner and recipes form 1`] = `
<div>
  <main
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      class="makeStyles-paper-1"
    >
      <div
        align="center"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
          role="progressbar"
          style="width: 40px; height: 40px;"
        >
          <svg
            class="MuiCircularProgress-svg"
            viewBox="22 22 44 44"
          >
            <circle
              class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
              cx="44"
              cy="44"
              fill="none"
              r="20.2"
              stroke-width="3.6"
            />
          </svg>
        </div>
      </div>
      <div
        align="center"
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-center"
      >
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <div
            class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
            role="progressbar"
            style="width: 40px; height: 40px;"
          >
            <svg
              class="MuiCircularProgress-svg"
              viewBox="22 22 44 44"
            >
              <circle
                class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                cx="44"
                cy="44"
                fill="none"
                r="20.2"
                stroke-width="3.6"
              />
            </svg>
          </div>
        </div>
      </div>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained"
        data-cy="asign-recipe-submit"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Asign Recipe
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </main>
</div>
`;
